TirusTimes/tirus-times-front
# chore: add Sentry to project

## Description

Add Sentry to project
Config Routes and loggers

## Related Issues

Use keywords like 'close' or 'solves' to link this PR to an issue.
For example:

Closes #12345
Unblocks #54321

## Progress

Use checkboxes to show that you did and what you have to do. For example:

- [x] Add Sentry to project;
- [x] Config Routes and loggers;

## How to test it

1 - yarn config-env
2 - yarn
3 - yarn dev
4 - Create an error
5 - Access Sentry to verify that error log is there 

## Visual reference

![image](https://user-images.githubusercontent.com/29777650/204096956-33dbb393-f417-48e3-8d07-d897dd15d350.png)

![image](https://user-images.githubusercontent.com/29777650/204096974-f777e167-4634-415e-b05b-6c248c6f5864.png)


## Change logs

Add Sentry to project;
Config Routes and loggers;
